demo/src/pages/index.js

```diff
@@ -7,34 +7,62 @@ import {
   Icon,
   GlobalHeader,
   NewRelicLogo,
+  HamburgerMenu,
 } from '@newrelic/gatsby-theme-newrelic';
 
-const IndexPage = ({ data }) => (
-  <>
-    <GlobalHeader editUrl="https://github.com/newrelic/gatsby-theme-newrelic/tree/develop/demo/src/pages/index.js" />
-    <div
-      css={css`
-        margin: 0 auto;
-        padding: ${data.site.layout.contentPadding};
-        max-width: ${data.site.layout.maxWidth};
-      `}
-    >
-      <NewRelicLogo />
-      <div>
-        Check it out on <Icon name={Icon.TYPE.GITHUB} />
-      </div>
-      <h1>Hello, World</h1>
-      <p>This is a test</p>
-      <Button
-        onClick={() => console.log('IT IS NOT')}
-        variant={Button.VARIANT.PRIMARY}
-        size={Button.SIZE.LARGE}
+const IndexPage = ({ data }) => {
+  const [isHamburgerOpen, setIsHamburgerOpen] = useState(false);
+  return (
+    <>
+      <GlobalHeader editUrl="https://github.com/newrelic/gatsby-theme-newrelic/tree/develop/demo/src/pages/index.js" />
+
+      <header
+        css={css`
+          position: relative;
+          border-bottom: 1px solid var(--divider-color);
+          padding: 0 2rem;
+          width: 100vw;
+        `}
+      >
+        <div
+          css={css`
+            display: flex;
+            align-items: center;
+            justify-content: space-between;
+            height: var(--height-mobile-nav-bar);
```

Comment:
FYI, this CSS variable is not available globally from the theme since this may or may not vary based on the site its used. This is the demo page, so thats ok, but just be aware that this height currently isn't doing anything.